core/orch_test.go

```diff
@@ -1320,8 +1352,36 @@ func TestPriceInfo_ReturnsBigRat(t *testing.T) {
 	expPricePerPixel = big.NewRat(20, 1)
 
 	priceInfo, err = orch.PriceInfo(ethcommon.Address{})
-	assert.Nil(t, err)
-	assert.Zero(t, expPricePerPixel.Cmp(big.NewRat(priceInfo.PricePerUnit, priceInfo.PixelsPerUnit)))
+	assert.Nil(err)
+	assert.Zero(expPricePerPixel.Cmp(big.NewRat(priceInfo.PricePerUnit, priceInfo.PixelsPerUnit)))
+	fixedPrice, err = common.PriceToFixed(expPricePerPixel)
+	expPrice = common.FixedToPrice(fixedPrice)
+	require.Nil(err)
+	assert.Equal(priceInfo.PricePerUnit, expPrice.Num().Int64())
+	assert.Equal(priceInfo.PixelsPerUnit, expPrice.Denom().Int64())
+
+	// test no overflows
+	basePrice = big.NewRat(25000, 1)
+	n.SetBasePrice(basePrice)
+	faceValue, _ := new(big.Int).SetString("22245599237119512", 10)
+	txCost := new(big.Int).Mul(big.NewInt(100000), big.NewInt(7500000000))
+	txMultiplier = new(big.Rat).SetFrac(faceValue, txCost) // 926899968213313/31250000000000
+	recipient = new(pm.MockRecipient)
+	n.Recipient = recipient
+	recipient.On("TxCostMultiplier", mock.Anything).Return(txMultiplier, nil)
+	orch = NewOrchestrator(n, nil)
+	overhead := new(big.Rat).Add(big.NewRat(1, 1), new(big.Rat).Inv(txMultiplier))
+	expPricePerPixel = new(big.Rat).Mul(basePrice, overhead) // 23953749205332825000/926899968213313
+	require.Equal(expPricePerPixel.Num().Cmp(big.NewInt(int64(math.MaxInt64))), 1)
+	priceInfo, err = orch.PriceInfo(ethcommon.Address{})
+	assert.Nil(err)
+	// for this case price will be rounded when converting to fixed
+	assert.NotEqual(expPricePerPixel.Cmp(big.NewRat(priceInfo.PricePerUnit, priceInfo.PixelsPerUnit)), 0)
+	fixedPrice, err = common.PriceToFixed(expPricePerPixel)
+	expPrice = common.FixedToPrice(fixedPrice)
+	require.Nil(err)
```

Comment:
`require` should be before previous line.

Reply:
fixed in 1ff75f7